PixelToolkit.py

```diff
@@ -11,13 +11,29 @@
 
     pass_gen = subparsers.add_parser("pass-gen", help="Password Generator")
     pass_gen.add_argument("--length", "-l", type=int, help="Length of the password", default=10)
+    
+    port_scan = subparsers.add_parser("port-scan", help="Port Scanner")
+    port_scan.add_argument("--host", type=str, help="Host to scan", default="127.0.0.1")
+    port_scan.add_argument("--ports", "-p", type=str, dest="port_range", default="1-65535", help="Port range to scan. Default is from 1 to 65535 (every port)")
 
     args = parser.parse_args()
-
     if len(sys.argv) == 1:  # If no arguments passed -> GUI
         from lib.gui import main_window_generator
         main_window_generator()
     else:
         if args.subcommand == "pass-gen":
             generator = PasswordGenerator()
             print("Password: " + generator.gen(args.length))
+
+        elif args.subcommand == "port-scan":
+            print(args)
```

Comment:
debugging is cool but we don't need that anymore

Reply:
sry, I forgot